### PR TITLE
Next new novel navigation nudge note 

### LINKED ISSUE #6402: New message: Inform wcadmin users about the new navigation
**Description:** we're going to test trigger an inbox notification targeting wcadmin users with the latest version of the new navigation. Our goal is to collect feedback as soon as possible about the latest improvements. 

**Title:** TBD
**Copy:** TBD
**Segmentation:** users with wcadmin 2.1 installed
**Timing:** once they update wcadmin to the 2.1 version
**Cadence:** Once
**CTA leads to:** wp-admin/admin.php?page=wc-settings&tab=advanced&section=features
**CTA label:** TBD

### PR DESCRIPTION
**Pull request by joshuatf**
Fixes #6402 

Adds the latest nav note to nudge users to enable the new nav feature.

### Screenshots
Screen Shot 2021-03-17 at 8 53 43 AM

### Detailed test instructions:

1. Download a plugin like WP Crontrol.
2. Turn on the new navigation under WooCommerce -> Settings -> Advanced.
3. Under Tools -> Cron Events, run the `wc_admin_daily` event.
4. Go to the homescreen, and note that the new note `You now have access to the WooCommerce navigation` is not added.
5. Turn off the new navigation.
6. Run `wc_admin_daily` again.
7. Go to the homescreen and see the new note.
8. Click `Enable` on the note actions.
9. Make sure the navigation has been enabled and you are redirected back to the screen where you viewed the note.
10. Delete the note if you have database access (or create a new site).
11. With navigation disabled, run the `wc_admin_daily` cron event.
12. Check that the note was added.
13. Manually turn on the note via WooCommerce -> Settings -> Advanced.
14. Check that the note was removed (actioned).

### CODE REVIEW

File: src/Notes/NavigationNudge.php [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+class NavigationNudge {
+	/**
+	 * Note traits.
+	 */
+	use NoteTraits;
+
+	/**
+	 * Name of the note for use in the database.
+	 */
+	const NOTE_NAME = 'wc-admin-navigation';
```
Comment (joelclimbsthings) [inline src/Notes/NavigationNudge.php:27]: What do you think of a slightly more specific name, `wc-admin-navigation-nudge` or something similar.

**Review by joelclimbsthings — DISMISSED** [review]
Having a little trouble testing. I activated a similar note, but it didn't have an _Enable_ button: 

image

File: src/Notes/NavigationNudge.php (hunk 1 above)
Comment (joshuatf) [reply src/Notes/NavigationNudge.php:27]: Good call.

**Comment by joshuatf** [comment]
Thanks for taking the time to review, @joelclimbsthings!

>Having a little trouble testing. I activated a similar note, but it didn't have an Enable button:

I think this may have been related to a previous note, I wonder if that note name was previously taken.  If so, the new note name should fix this.

**Comment by joelclimbsthings** [comment]
Hey Josh, thanks for the change. I'm still having some issues getting this to work as expected. Here's what's happening on my end, which I observed consistently on two separate fresh sites:

- Create site, install WC, and this branch of wc-admin, crontrol
- Enable new navigation
- Run `wc_admin_daily` event
- Go back to home, and now I see the other _similar_ note that I included a screenshot of in my last comment. It looks like this note has the name `wc-admin-navigation-feedback`, so it wasn't really a conflict.
- Disable new navigation
- Go back and run the `wc_admin_daily` event
- Go back to home, and I don't see any changes (including no sign of your new nudge note)

**Comment by joshuatf** [comment]
Hey @joelclimbsthings - sorry about the confusion.  There was an error happening due to the renamed class.  Updated that and tested a fresh site.

**Comment by joshuatf** [comment]
I just realized this can bypass the usage tracking this way.  Do we just want to redirect them to the settings page and let them opt in from there?  cc @pmcpinto @elizaan36

**Comment by elizaan36** [comment]
Ah, yep we were going to redirect people to Settings to enable the new nav from the Inbox note. That way they'll go through the data tracking flow. Could change the CTA to "Enable in Settings" to make sure it's clear what will happen.

**Comment by joshuatf** [comment]
Thanks, @elizaan36 - updated the CTA text as well as the action here.

@joelclimbsthings This is ready for another review.  Note I also updated some antiquated logic in https://github.com/woocommerce/woocommerce-admin/pull/6610/commits/8435db6bc4058645a13bdf6db2debecb5bdad629 that prevented the check of existence of the nav feature.

**Comment by joelclimbsthings** [comment]
Great @joshuatf . This now works beautifully. :shipit:

**Review by psealock — APPROVED** [review]
Based on this and the label "Ready to merge" I'll give approval

>This now works beautifully. :shipit:

**Pull request merged**